utils/swift_build_support/swift_build_support/products/sourcekitlsp.py

```diff
@@ -73,3 +89,54 @@ def get_dependencies(cls):
                 xctest.XCTest,
                 llbuild.LLBuild,
                 swiftpm.SwiftPM]
+
+    def run_build_script_helper(self, action, base_target, host_target):
+        # base_target is the machine that's driving the build.
+        # host_target is the target we are bulding for.
+        script_path = os.path.join(
+            self.source_dir, 'Utilities', 'build-script-helper.py')
+
+        install_destdir = self.host_install_destdir(host_target)
+        toolchain_path = self.native_toolchain_path(base_target)
```

Comment:
`base_target` parameter is only used for this. I'm not really sure what's the semantics of `native_toolchain_path` argument. Deferring to @edymtt .

Also, I think
> ```
> # base_target is the machine that's driving the build.
> ```
is only true for Darwin platforms (I think) For other platforms, `build`/`installed` is called for each host. Should we use `self.arg.host-target` instead?